develop/styles/shared/customizations.scss

```diff
@@ -8,3 +10,17 @@
         }
     }
 }
+
+/* Utility classes */
+.text-xs {
```

Comment:
i dont think you need this, those variables are available by importing "~@sebgroup/bootstrap/scss/_variables" the _variable file is already importing  "bootstrap-core/variables" line 537